rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -127,6 +157,11 @@ function _M.call()
     return
   end
 
+  if starts_with(ngx.var.request_uri, "/configuration/certs") then
```

Comment:
Maybe use `ngx.var.uri` instead of using `starts_with`?

Reply:
fixed